cross-platform/react-app/package.json

```diff
@@ -9,9 +9,9 @@
   "private": true,
   "homepage": "./",
   "scripts": {
-    "start": "USE_FAST_SASS=true BROWSER=none react-scripts start",
+    "start": "GENERATE_SOURCEMAP=false USE_FAST_SASS=true BROWSER=none react-scripts start",
```

Comment:
you migth also want to set these two for start & build

```
TRANSPILE_DEPS=false
DISABLE_TERSER=true
```

Reply:
Can you explain what those do?

Reply:
https://github.com/imodeljs/create-react-app/blob/imodeljs/packages/react-scripts/README-imodeljs.md

Reply:
Updated.